crt_portal/cts_forms/views.py

```diff
@@ -301,8 +301,11 @@ def index_view(request):
             p_class_list = p_class_list[:3]
             p_class_list[2] = f'{p_class_list[2]}...'
 
+        email_report_count = Report.objects.filter(contact_email=report.contact_email).count()
```

Comment:
Seems a bit slow to me to do this in a loop especially if the `per_page` QP is set to a high number (normally it defaults to 15).

Is the intention to get this in there and optimize later?

Reply:
Also, I know you already spoke with Emily but are we going to adjust this PR to display `-` if there's no email or do this in a different PR?

Reply:
Thanks for reviewing!

1. Agreed that this is not optimal. I fiddled a little but couldn't immediately figure out how to `aggregate/annotate` the `Report` query such that each record includes the total count of records with the same `contact_email`. The thinking here is that (at least) this part of the implementation will have to be refactored anyway when we incorporate sorting, which I presume will happen pretty much immediately after this gets merged. So this was more of a quick & dirty solution to at least get the counts displaying as a part of this sprint. 
2. Yes, I'm going to mark this PR as WIP until I hear from Emily about what she wants to do in the email=None case.

Reply:
The aggregation I landed on when fiddling with this a little bit was something like...count just the contact_emails we're about to render w/
```python
    emails_in_reports_on_this_page = [r.contact_email for r in requested_reports]
    emails_counts = Report.objects.filter(contact_email__in=emails_in_reports_on_this_page).values('contact_email').annotate(total=Count('contact_email'))
```

1 more query per page -Returns a queryset like which could be mapped back onto Reports as their being passed to context
```
<QuerySet [{'contact_email': 'testing@test.edu', 'total': 19}, {'contact_email': 'testing@test.test', 'total': 11}]>
```